### PR TITLE
Feature/update

### PR DESCRIPTION
**Pull request by gersonhiggins**
Transacction 1:
image
Transacction 2:
image
Transacction 3:
image
Transacction 4:
image
image
queries: 
image

Merging Feature/update into develop
-playing with transaction and queries exersice

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,4 +4,10 @@ INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_
...
 INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (4, 'Devimon', '12/05/2017', 5, true, 11);
-
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (5, 'Charmander', '2020-02-08', 0, false, -11);
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (6, 'Plantmon', '2021-11-15', 2, true, -5.7);
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (7, 'Squirtle', '1993-04-02', 3, false, -12.13);
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (8, 'Angemon', '2005-06-12', 1, true, -45);
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (9, 'Boarmon', '2005-06-07', 7, true, 20.4);
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (10, 'Blossom', '1998-10-13', 3, true, 17);
+INSERT INTO animals (ID, NAME, DATE_OF_BIRTH, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (11, 'Ditto', '2022-05-14', 4, true, 22);
```
Comment (iambenkis) [inline data.sql:13]: - [ ] Keep doing the great work 😄

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @gersonhiggins  

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Data were added 
✔️ Schema was respected  
✔️ Screenshot with correct output

## Require Changes ♻️ 

- [ ] Kindly remove the `Note`in your README file, it does not contain important information for visitors 👍 of your account. It will make your documentation more professional  😄
 
Capture

- [ ] You did not use React or Express.js anywhere in your project , but I put them in the documentation ,always make sure you include the tech stack you used in your project 👍 for more professionalism 🛩️ 

Capture



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 2]
```diff
@@ -239,6 +225,4 @@ I would like to thank... Microverse
```
Comment (Ismailco) [inline README.md:225]: - [ ] [OPTIONAL] The `issues` link is broken, kindly make sure to fix it so your documentation is professional

**Review by Ismailco — APPROVED** [review]
### Hi @gersonhiggins, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**